<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OSRM_Table_DialogBase</class>
 <widget class="QDialog" name="OSRM_Table_DialogBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>448</width>
    <height>309</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>OSRM: Solve the TSP with OSRM</string>
  </property>
  <widget class="QPushButton" name="pushButton_about">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>267</y>
     <width>85</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>About..</string>
   </property>
  </widget>
  <widget class="QDialogButtonBox" name="close_button_box">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>267</y>
     <width>91</width>
     <height>30</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Close</set>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_display">
   <property name="geometry">
    <rect>
     <x>23</x>
     <y>226</y>
     <width>411</width>
     <height>30</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>9</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Display the result</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_clear">
   <property name="geometry">
    <rect>
     <x>23</x>
     <y>189</y>
     <width>410</width>
     <height>30</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Clear previous results</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_provider">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>6</y>
     <width>281</width>
     <height>35</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>9</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>11</y>
     <width>131</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Select Provider:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>56</y>
     <width>251</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Select layer with points to visit:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>111</width>
     <height>24</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Source point layer:</string>
   </property>
  </widget>
  <widget class="QgsMapLayerComboBox" name="comboBox_layer">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>90</y>
     <width>281</width>
     <height>32</height>
    </rect>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_selec_features">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>130</y>
     <width>433</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Only between selected features</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_instructions">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>160</y>
     <width>261</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>9</pointsize>
     <italic>false</italic>
    </font>
   </property>
   <property name="text">
    <string>Display routing instructions</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QgsMapLayerComboBox</class>
   <extends>QComboBox</extends>
   <header>qgsmaplayercombobox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>close_button_box</sender>
   <signal>accepted()</signal>
   <receiver>OSRM_Table_DialogBase</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>close_button_box</sender>
   <signal>rejected()</signal>
   <receiver>OSRM_Table_DialogBase</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
